### PR TITLE
Added instruction to perform runtime upgrade

### PR DESCRIPTION
**Pull request by ayushmishra2005**

### CODE REVIEW

File: docs/tutorials/tutorials-runtime-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## How to perform runtime-upgrade
+
+* Clone <a href="https://github.com/DataHighway-DHX/node" target="_blank" class="pretty-link pretty-link-colored">node</a> repo
+
+* `cd node`
+
+* `cargo build --release`
+
+* Import the Sudo account on the https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwestlake.datahighway.com#/accounts
```
Comment (ltfschoen) [inline docs/tutorials/tutorials-runtime-upgrade.md:15]: the community who will be using documentation published at https://dev.datahighway.com/ won't have access to Sudo. they'll need guidance on how to propose changes to the runtime that'd be voted upon by the community through on-chain governance.

File: docs/tutorials/tutorials-runtime-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+id: tutorials-runtime-upgrade
+title: Runtime Upgrade
+sidebar_label: Runtime Upgrade
+---
+
+## How to perform runtime-upgrade
```
Comment (ltfschoen) [inline docs/tutorials/tutorials-runtime-upgrade.md:7]:
```suggestion
## How to propose a runtime-upgrade
```

File: docs/tutorials/tutorials-runtime-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+* Choose _sudo_ as extrinsic and select _sudoUncheckedWeight_ as function
+
+* Choose _system_ as call and _setCode_ as function 
+
+* Click the _code_ input field, and select the Wasm binary that defines the upgraded runtime: 
+  `target/release/wbuild/datahighway-runtime/datahighway_runtime.compact.wasm`
+  
+* Copy the preimage hash. This represents the proposal. You will use this hash when submitting the actual proposal.
+
+* Leave the value for the _weight parameter at the default of 0. Click "Submit Transaction" and then "Sign and Submit".
```
Comment (ltfschoen) [inline docs/tutorials/tutorials-runtime-upgrade.md:32]: this should be in Step 4: Submit the Preimage of the Proposal for runtime-upgrade
```suggestion
```

File: docs/tutorials/tutorials-runtime-upgrade.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+### Step 1: Prepare WASM binary 
+
+* Clone <a href="https://github.com/DataHighway-DHX/node" target="_blank" class="pretty-link pretty-link-colored">node</a> repo
+
+* `cd node`
+
+* `cargo build --release`
+
+### Step 2: Submit a Preimage of the Proposal for runtime-upgrade
```
Comment (ltfschoen) [inline docs/tutorials/tutorials-runtime-upgrade.md:17]:
```suggestion
### Step 2: Generate a Preimage Hash of the Proposal for runtime-upgrade
```

File: docs/tutorials/tutorials-runtime-upgrade.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+### Step 3: Submit the Proposal
+
+* Select the account from which you want to submit the proposal. 
+
+* Enter the preimage hash related to the proposal. 
+
+* Set the locked balance. 
+
+* Click the _Submit proposal_ button and sign the transaction
```
Comment (ltfschoen) [inline docs/tutorials/tutorials-runtime-upgrade.md:42]:
```suggestion
* Click the _Submit proposal_ button and sign the transaction

### Step 4: Submit the Preimage of the Proposal for runtime-upgrade

* Repeat Step 2.

* Leave the value for the _weight parameter at the default of 0.

* Click "Submit Transaction" and then "Sign and Submit".
```

File: docs/tutorials/tutorials-runtime-upgrade.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+sidebar_label: Runtime Upgrade
+---
+
+## How to perform runtime-upgrade
+
+### Step 1: Prepare WASM binary 
+
+* Clone <a href="https://github.com/DataHighway-DHX/node" target="_blank" class="pretty-link pretty-link-colored">node</a> repo
+
+* `cd node`
```
Comment (ltfschoen) [inline docs/tutorials/tutorials-runtime-upgrade.md:13]:
```suggestion
* Switch to the root directory of the cloned project and build the Wasm bytecode
```bash
cd node
cargo build --release
```
```

File: docs/tutorials/tutorials-runtime-upgrade.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
```
Comment (ltfschoen) [inline docs/tutorials/tutorials-runtime-upgrade.md:14]:
```suggestion
```

File: docs/tutorials/tutorials-runtime-upgrade.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+* `cargo build --release`
```
Comment (ltfschoen) [inline docs/tutorials/tutorials-runtime-upgrade.md:15]:
```suggestion
```

Comment (ltfschoen) [inline docs/tutorials/tutorials-runtime-upgrade.md:15]:
```suggestion
* Build the Wasm bytecode
```
cargo build --release
```
```

**Review by ltfschoen — CHANGES_REQUESTED** [review]
proposed changes shown.

File: docs/tutorials/tutorials-runtime-upgrade.md (hunk 3 above)
Comment (ayushmishra2005) [reply docs/tutorials/tutorials-runtime-upgrade.md]: No. This is required at step 2. I already verified on local.

**Comment by ayushmishra2005** [comment]
> proposed changes shown.

@ltfschoen   All comments have been addressed. Please review

File: docs/tutorials/tutorials-runtime-upgrade.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+* Switch to the root directory of the cloned project
+
+`cd node`
+
+* Build the Wasm bytecode
+
+`cargo build --release`
+
+
+### Step 2: Generate a Preimage Hash of the Proposal for a forkless runtime upgrade
```
Comment (ltfschoen) [inline docs/tutorials/tutorials-runtime-upgrade.md:22]:
```suggestion
### Step 2: Submit a Preimage Hash of the Proposal for a forkless runtime upgrade
```

**Review by ltfschoen — APPROVED** [review]

**Pull request merged**